### ISSUE TITLE
Blank report

### ISSUE THREAD

**Issue by klustafuc** [alanrenouf/vCheck-vSphere#346]
Hi,

When i run your script it takes a long time to gather all the information and there are a few expected errors, yet the output is blank and the attachment is 1KB completely empty..

The files are unblocked and accessible.

**Comment by Sneddo** [comment]
Can you provide the errors you get? Also PowerCLI/vCheck and vCenter versions.

**Comment by klustafuc** [comment]
PowerCLI 5.5 release 1 , vCenter 5.5 U1 and vCheck is the latest

Method invocation failed because [System.String] doesn't contain a method named 'RemoveChild'.
At C:\Scripts\vCheck-vSphere-master\vCheck.ps1:203 char:31
+    $XMLTable.table.RemoveChild <<<< ($XMLTable.table.colgroup) | out-null
    + CategoryInfo          : InvalidOperation: (RemoveChild:String) [], RuntimeException
    + FullyQualifiedErrorId : MethodNotFound

Method invocation failed because [System.String] doesn't contain a method named 'SetAttribute'.
At C:\Scripts\vCheck-vSphere-master\vCheck.ps1:204 char:32
+    $XMLTable.table.SetAttribute <<<< ("width","100%")
    + CategoryInfo          : InvalidOperation: (SetAttribute:String) [], RuntimeException
    + FullyQualifiedErrorId : MethodNotFound

The term 'Get-ReportHTML' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path i
s correct and try again.
At C:\Scripts\vCheck-vSphere-master\vCheck.ps1:720 char:27
+ $MyReport = Get-ReportHTML <<<<
    + CategoryInfo          : ObjectNotFound: (Get-ReportHTML:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

Many Thanks

**Comment by Sneddo** [comment]
You need to update your style for a start, that should get you some output. 

I do wonder why you are getting table formatting issues though - which plugin did these show up after running?

**Comment by Sneddo** [comment]
No update in 3 weeks, will close this off. Log another issue if you are still having trouble.
